py/desiutil/test/test_bitmask.py

```diff
@@ -105,6 +106,29 @@ def test_str(self):
             self.assertEqual(bitmask[name].bitnum, self.ccdmask[name].bitnum)
             self.assertEqual(bitmask[name]._extra, self.ccdmask[name]._extra)
 
+    def test_uint64(self):
+        _bitdefs = dict(ccdmask=list())
+        _bitdefs['ccdmask'].append( ['BAD',   0, "badness"] )
+        _bitdefs['ccdmask'].append( ['HOT',   1, "hothot"] )
+        _bitdefs['ccdmask'].append( ['TEST', 16, "testing"] )
+
```

Comment:
Yes, you will certainly have problems in Python 2.  Instead of commenting out the test, I would add it as a skipped test or an expected failure under Python 2.  Technically under Python 3.5, the integers are not "gracefully becoming long".  There is no `long()` in Python 3.  All integers in Python 3 are what used to be called long in Python 2.